project/src/store/api-actions.test.js

```diff
@@ -0,0 +1,813 @@
+          goods:
+            ['Towels', 'Fridge', 'Air conditioning', 'Washing machine', 'Breakfast', 'Dishwasher', 'Laptop friendly workspace', 'Coffee machine', 'Washer', 'Baby seat'],
+          host:
+          {
+            avatar_url: 'img/avatar-angelina.jpg',
+            id: 25,
+            is_pro: true,
+            name: 'Angelina',
+          },
+          id: 1,
+          images:
+            ['https://7.react.pages.academy/static/hotel/12.jpg', 'https://7.react.pages.academy/static/hotel/18.jpg', 'https://7.react.pages.academy/static/hotel/20.jpg', 'https://7.react.pages.academy/static/hotel/13.jpg', 'https://7.react.pages.academy'],
+
+          is_favorite: false,
+          is_premium: false,
+          location:
+          {
+            latitude: 48.83961,
+            longitude: 2.342499,
+            zoom: 16,
+          },
+          max_adults: 8,
+          preview_image: 'https://7.react.pages.academy/static/hotel/15.jpg',
+          price: 291,
+          rating: 4.3,
+          title: 'Penthouse, 4-5 rooms + 5 balconies',
+          type: 'hotel',
+        },
+        {
+          bedrooms: 3,
+          city:
+          {
+            location:
+            {
+              latitude: 48.85661,
+              longitude: 2.351499,
+              zoom: 13,
+            },
+            name: 'Paris',
+          },
+          description: 'I rent out a very sunny and bright apartment only 7 minutes walking distance to the metro station. The apartment has a spacious living room with a kitchen, one bedroom and a bathroom with mit bath. A terrace can be used in summer.',
+          goods:
+            ['Towels', 'Fridge', 'Air conditioning', 'Washing machine', 'Breakfast', 'Dishwasher', 'Laptop friendly workspace', 'Coffee machine', 'Washer', 'Baby seat'],
+          host:
+          {
+            avatar_url: 'img/avatar-angelina.jpg',
+            id: 25,
+            is_pro: true,
+            name: 'Angelina',
+          },
+          id: 1,
+          images:
+            ['https://7.react.pages.academy/static/hotel/12.jpg', 'https://7.react.pages.academy/static/hotel/18.jpg', 'https://7.react.pages.academy/static/hotel/20.jpg', 'https://7.react.pages.academy/static/hotel/13.jpg', 'https://7.react.pages.academy'],
+
+          is_favorite: false,
+          is_premium: false,
+          location:
+          {
+            latitude: 48.83961,
+            longitude: 2.342499,
+            zoom: 16,
+          },
+          max_adults: 8,
+          preview_image: 'https://7.react.pages.academy/static/hotel/15.jpg',
+          price: 291,
+          rating: 4.3,
+          title: 'Penthouse, 4-5 rooms + 5 balconies',
+          type: 'hotel',
+        }],
+      );
+
+    return fetchHotelsLoader(dispatch, () => { }, api)
+      .then(() => {
+        expect(dispatch).toHaveBeenCalledTimes(1);
+        expect(dispatch).toHaveBeenNthCalledWith(1, {
+          type: ActionType.SET_OFFERS,
+          payload: offers,
```

Comment:
```suggestion
          payload: adaptToClient(offers),
```
offers - массив того вида, в котором они приходят с сервера